optuna_dashboard/ts/action.ts

```diff
@@ -251,14 +256,20 @@ export const actionCreator = () => {
       nLocalFixedTrials =
         firstUpdatable === -1 ? currentTrials.length : firstUpdatable
     }
-    getStudyDetailAPI(studyId, nLocalFixedTrials)
+    getStudyDetailAPI(studyId, nLocalFixedTrials, nMaximumTrialsAtOnce)
       .then((study) => {
+        if (studyId in studyDetails && study.trials.length === 0) {
+          // Update trials only if necessary.
+          // NOTE: The first condition is for study with no trials.
+          return
+        }
```

Comment:
Does this change support the following scenarios?

1. Creating an Optuna study (e.g. `$ optuna create-study ...`)
2. Starting the Optuna Dashboard and opening the URL in a web browser.
3. Loading an Optuna study and starting the optimization process (e.g. `study = optuna.load_study(...); study.optimize(...)`.

Reply:
It works in my environment!